services/slack/service.go

```diff
@@ -99,7 +119,8 @@ func (s *Service) Alert(channel, message, username, iconEmoji string, level aler
 	if err != nil {
 		return err
 	}
-	resp, err := http.Post(url, "application/json", post)
+	s.logger.Println("D! Posting event to Slack", post)
+	resp, err := s.client.Post(url, "application/json", post)
```

Comment:
This line races with the lines in Update and New that assign to the client. Either add a mutex to the Sensu service or use the sync.Atomic type to synchronizes access to the client.

Other than that this looks good! Thanks